tasks/aliases.yaml

```diff
@@ -5,6 +5,7 @@ test:
   - connect:serve
   - sauce_connect
   - protractor
```

Comment:
I think you did, but as I also modified aliases.yaml in master, probably it has get back again during merge. Sorry!